packages/perseus-core/src/data-schema.ts

```diff
@@ -1260,7 +1262,7 @@ export type PerseusNumberLineWidgetOptions = {
     // This controls the number (and position) of the tick marks; you can either set the number of divisions (2 divisions would split the entire range in two halves), or the tick step (the distance between ticks) and the other value will be updated accordingly. Note:  There is no check to see if labels coordinate with the tick marks, which may be confusing for users if the blue labels and black ticks are off-step.
     tickStep?: number | null;
     // The correct relative value. default: "eq". options: "eq", "lt", "gt", "le", "ge"
```

Comment:
Can you also remove the part of the comment that defines the options? I find these types of comments are just ripe to age poorly and become inaccurate. Plus, with the addition of the actual union definition, it becomes extremely redundant.